src/jaspContainer.h

```diff
@@ -6,6 +6,7 @@
 #include "jaspState.h"
 #include "jaspHtml.h"
 #include "jaspQmlSource.h"
+#include "jaspReport.h"
```

Comment:
deze wordt toch niet gebruikt?

Reply:
zie https://github.com/jasp-stats/jaspBase/pull/100#discussion_r1022825725 

maar ook zoiezo voor jaspResults moet de klasse ergens worden geinclude en aangezien results kind is van container is dit dus een prima plek.